scopes/ui-foundation/ui/pre-bundle-preview.task.ts

```diff
@@ -0,0 +1,67 @@
+import { join } from 'path';
+import { existsSync, mkdirSync, writeFileSync } from 'fs';
+import { BuildContext, BuildTask, BuiltTaskResult, TaskLocation } from '@teambit/builder';
+import { Capsule } from '@teambit/isolator';
+import { Logger } from '@teambit/logger';
+import { UIAspect, UiMain } from '@teambit/ui';
+
+export const PRE_BUNDLE_PREVIEW_TASK_NAME = 'PreBundlePreview';
+export const PRE_BUNDLE_PREVIEW_DIR = 'preview-pre-bundle';
+export const PRE_BUNDLE_PREVIEW_HASH_FILENAME = '.hash';
+
+export class PreBundlePreviewTask implements BuildTask {
+  aspectId = 'teambit.ui-foundation/preview';
```

Comment:
once moved the file to preview aspect, take the aspect id from the `preview.aspect.ts` file